doc/docs.md

```diff
@@ -4121,6 +4122,19 @@ A vfmt run is usually pretty cheap (takes <30ms).
 
 Always run `v fmt -w file.v` before pushing your code.
 
+### v shader
+
+You can use GPU shaders with V graphical apps. You write your shaders in an
+[annotated GLSL dialect](https://github.com/vlang/v/blob/1d8ece7/examples/sokol/02_cubes_glsl/cube_glsl.glsl)
+and use `v shader` to compile them for all supported target platforms.
+
+```shell
+v shader /path/to/project/dir/or/file.v
+```
+
+Currently you need to include a header and delcare a glue function before
+using the shader in your code.
+
```

Comment:
Good to know... but could you point to an example?

Reply:
Good point, will add when I  have time 👍

Reply:
Added in #12613